eng/common/scripts/Prepare-Release.ps1

```diff
@@ -2,13 +2,18 @@
 
 [CmdletBinding()]
 param(
-    [Parameter(Mandatory=$true)]
     [string]$PackageName,
-    [string]$ServiceDirectory,
+    [string]$ArtifactName,
```

Comment:
I don't think we need to care about the artifact name in this scenario. I think we should just require this be the package name all the time.